rest_framework/serializers.py

```diff
@@ -900,6 +900,35 @@ class ModelSerializer(Serializer):
     # "HTTP 201 Created" responses.
     url_field_name = None
 
+    def set_meta_fields(self,lst):
```

Comment:
you need tests & docs to make this PR a feasible one. I am not sure it this is an intended design for the framework maintainers